index.html

```diff
@@ -1,8 +1,92 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
-  <head>
-    <title>Memory exercise</title>
-  </head>
-  <body>
-  </body>
+<head>
+<meta charset="UTF-8">
+<title>New Memory Game</title>
+<script>
+var back = 'back.gif';
+var tile = ['img0.gif','img1.gif','img2.gif','img3.gif','img4.gif','img5.gif',
+'img6.gif','img7.gif','img8.gif','img9.gif','img10.gif','img11.gif',
+'img12.gif','img13.gif','img14.gif'];
+
+function randOrd(a, b){
+	return (Math.round(Math.random())-0.5);
+	} 
+
+var im = []; 
+for (var i = 0; i < 15; i++) {
+	im[i] = new Image(); 
+	im[i].src = tile[i]; 
+	tile[i] = '<img src="'+tile[i]+'" width="60" height="60" alt="tile" \/>'; 
+	tile[i+15] = tile[i];
+	} 
+
+function displayBack(i) {
+	document.getElementById('t'+i).innerHTML = '<div onclick="disp('+i+');return false;"><img src="'+back+'" width="60" height="60" alt="back" \/><\/div>';
```

Comment:
Try to avoid doing inline JS – use [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) instead!

http://en.wikipedia.org/wiki/Unobtrusive_JavaScript